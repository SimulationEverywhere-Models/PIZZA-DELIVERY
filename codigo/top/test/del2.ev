00:00:00:004 Llamada_del 3
00:00:00:008 Llamada_del 4
00:00:08:000 Abrir_del 1
00:00:13:000 Llamada_del 2
00:00:16:000 Llamada_del 5
00:00:19:000 Llamada_del 6
00:00:22:000 Llamada_del 7
00:00:25:000 Cerrar_del 1
