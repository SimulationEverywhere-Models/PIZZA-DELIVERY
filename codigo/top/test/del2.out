00:00:00:008 ocupada_del 1
00:00:00:019 no_atiende_del 1
00:00:36:120 salida_del 6
00:00:36:240 salida_del 2
00:00:40:090 salida_del 7
00:00:40:450 salida_del 5
